internal/resourcepolicies/volume_resources.go

```diff
@@ -66,6 +69,19 @@ func (s *structuredVolume) parsePV(pv *corev1api.PersistentVolume) {
 	s.volumeType = getVolumeTypeFromPV(pv)
 }
 
+func (s *structuredVolume) parsePVC(secondaryRes any) error {
```

Comment:
Since the caller passes a PVC pointer (from `VolumeFilterData`) decisively, here we should have the parameter as `pvc *corev1api.PersistentVolumeClaim`, in this way, we won't need to return error for this function.

Reply:
Done, updated the PR.